src/main/resources/application.yaml

```diff
@@ -133,6 +134,7 @@ containers:
       jurisdiction: BULKSCAN
       poBox: BULKSCANPO
       ocrValidationUrl: ${OCR_VALIDATION_URL_BULKSCAN_SAMPLE_APP}
+      payments-enabled: true
```

Comment:
perhaps best to stick to common naming convention at least within same config scope even though i **do** prefer hyphens